code/mob/living/carbon/human/procs/damage.dm

```diff
@@ -4,8 +4,47 @@
 	if(!.)
 		return 0 //early return from parent
 
+<<<<<<< HEAD
```

Comment:
looks like you didn't resolve a merge conflict here but you marked it as resolved

Reply:
should be done